### DISCUSSION TITLE
[Feature Request]: Ability to create Boards with columns other than status

### DISCUSSION THREAD

**Discussion by JoshHrach** [community/community#6402] [Projects and Issues] [answered]
While I understand the usual importance of status columns in Board view, I like using Board view for visualizing other things. In Asana, I would use it to visualize what features are going into a particular planned release. In their board view, the columns let me easily drag and drop features between versions. In a list, it shows up as a "Single Select" field.

Since I moved to Github 2 years ago for my personal projects, I've manually managed columns to visualize features across versions. I would like to be able to do that with Boards here, but it seems like it's strongly tied to Status at present. Renaming the columns or adding to them just adds/updates the Status field options.

Please provide a way to create column views based on fields other than Status.

**Comment by alexcnichols** [comment]
👋🏼 Improvements for pivoting the board by other columns, and more, are coming soon! Check out this roadmap item for more and subscribe to be notified if/when things change. https://github.com/github/roadmap/issues/282 :octocat:

**Comment by azenMatt** [comment, marked as answer]
Hi @JoshHrach  - we just released this! 

Check it out [here](https://github.blog/changelog/2021-12-02-the-new-github-issues-12-02-update/) and let us know what you think!

**Comment by JoshHrach** [comment]
@azenMatt That's great! I just tested it out and it's nice to see this with the new projects. Excellent work.
